js/ui/validation_message.js

```diff
@@ -158,7 +158,7 @@ const ValidationMessage = Overlay.inherit({
             boundary,
             my: positions[0],
             at: positions[1],
-            collision: 'none flip'
```

Comment:
Why do u think we should change it? I think that, otherwise, collision should be specified for both directions

Reply:
>> Why do u think we should change it
Yes
![image](https://user-images.githubusercontent.com/29150145/179569617-5c202c0e-2db3-4f87-97a7-189afb8b5129.png)